packages/@aws-cdk/aws-codecommit/README.md

```diff
@@ -37,6 +37,30 @@ To add an Amazon SNS trigger to your repository:
 repo.notify('arn:aws:sns:*:123456789012:my_topic');
 ```
 
+## Initialize with Code
+
+It is possible to initialize the Repository with an Asset.
+For learning how assets work, see the CDK Assets documentation.
+Be aware, that this will only work on *directories*, not on single files.
```

Comment:
We should validate that we are passed the path to a directory then, and not a file.

Reply:
is invalid by now